internal/common/metrics.go

```diff
@@ -64,9 +47,9 @@ func sendStatsdMetrics(dataRate int, duration time.Duration) error {
 	for {
 		select {
 		case <-ticker.C:
-			for time := 0; time < dataRate; time++ {
+			for time := 0; time < metricPerMinute; time++ {
 				go func(time int) {
-					client.Inc(fmt.Sprintf("%v", time), int64(time), 1.0)
+					client.Inc(fmt.Sprintf("statsd_metric_%v", time), int64(time), 1.0)
 				}(time)
```

Comment:
Left it for now though. What do you all think?

Reply:
Yes that's why I generate go routine since stress are more than [50,000/minutes](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/generator/resources/ec2_stress_test_matrix.json#L43) and bench marks are more than `5,000/minutes`